src/main/java/core/basesyntax/service/operations/SupplyOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public void getOperation(List<FruitTransaction> transactions) {

```

Comment:
The method name `getOperation` is misleading as it does not actually get or return anything. A more appropriate name might be `apply` or `execute` to reflect the action it performs on the transactions.